mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -3705,7 +3705,7 @@ func TestAccContainerCluster_errorNoClusterCreated(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:		 testAccContainerCluster_withInvalidLocation("wonderland"),
-				ExpectError: regexp.MustCompile(`Location "wonderland" does not exist.`),
+				ExpectError: regexp.MustCompile(`(Location "wonderland" does not exist)|(Permission denied on 'locations\/wonderland' \(or it may not exist\))`),
```

Comment:
I just appended the new error here but I'm happy to swap over to just the new error message if that's clearer!

Reply:
This looks good to me! thanks!